### ISSUE TITLE
Feature Request: Provide validation as CLI tool

### ISSUE THREAD

**Issue by hbruch** [MobilityData/gbfs-validator-java#97]
The official GBFS validator https://gbfs-validator.netlify.app/ has issues validating larger GBFS feeds.

It would be nice, if `gbfs-validator-java` would provide an easy to use CLI version to validate such feeds.

**Comment by testower** [comment]
Indeed, there will be some preparatory work to accomodate this - see https://github.com/entur/gbfs-validator-java/issues/83
